src/xrc/src/api.rs

```diff
@@ -396,6 +399,10 @@ enum ValidateRequestError {
     AlreadyInflight,
     /// The request timestamp that goes back in the past is not in the rate cache.
     PastTimestampNotCached,
+    CryptoBaseAssetEmptySymbol,
+    FiatBaseAssetEmptySymbol,
+    CryptoQuoteAssetEmptySymbol,
+    FiatQuoteAssetEmptySymbol,
```

Comment:
Is it useful to distinguish between `CryptoBaseAssetEmptySymbol` and `FiatBaseAssetEmptySymbol`?
Wouldn't `BaseAssetEmptySymbol` and `QuoteAssetEmptySymbol` be enough?

Reply:
It's useful in the sense of mapping the error to the `CryptoBaseAssetNotFound`, etc. If we want, we could simply this down to just use the `Other` error type and adding a new error type. It just seemed friendlier to match up the errors.